render/persistentvolume.go

```diff
@@ -0,0 +1,90 @@
+package render
+
+import (
+	"github.com/weaveworks/scope/probe/kubernetes"
+	"github.com/weaveworks/scope/report"
+)
+
+// KubernetesVolumesRenderer renders Kubernetes volume components
+var KubernetesVolumesRenderer = MakeReduce(
+	VolumesRenderer(),
+	PodToVolumeRenderer(),
+	PVCToStorageClassRenderer(),
+)
+
+// VolumesRenderer returns renderer
+func VolumesRenderer() Renderer {
+	return volumesRenderer{}
+}
+
+// volumesRenderer is the renderer to render PVC & PV
+type volumesRenderer struct{}
+
+// Render renders the nodes
+func (v volumesRenderer) Render(rpt report.Report) Nodes {
+	nodes := make(report.Nodes)
+	for id, n := range rpt.PersistentVolumeClaim.Nodes {
+		volume, _ := n.Latest.Lookup(kubernetes.VolumeName)
+		for pvNodeID, p := range rpt.PersistentVolume.Nodes {
+			volumeName, _ := p.Latest.Lookup(kubernetes.Name)
+			if volume == volumeName {
+				n.Adjacency = n.Adjacency.Add(p.ID)
+				n.Children = n.Children.Add(p)
+			}
+			nodes[pvNodeID] = p
+		}
+		nodes[id] = n
+	}
+	return Nodes{Nodes: nodes}
+}
+
+//PodToVolumeRenderer renders Pod and PVC resources
+func PodToVolumeRenderer() Renderer {
+	return podToVolumesRenderer{}
+}
+
+// VolumesRenderer is the renderer to render volumes
+type podToVolumesRenderer struct{}
+
+// Render renders the nodes
```

Comment:
This comment is not very helpful - "nodes" is a very generic term.  Say a little about what is really being done.